src/components/Profile.tsx

```diff
@@ -80,7 +81,7 @@ const Profile = () => {
 
               <div className="flex gap-2 align-middle text-mid">
                 <a
-                  href={`https://viewblock.io/arweave/address/${walletAddress.toString()}`}
+                  href={getBlockExplorerUrlForAddress(walletAddress.toString())}
```

Comment:
should probably add this to wallet connector

Reply:
I did the block explorer per connector in Turbo Topup, but I based this connector code in what was in ArNS. Could look at it for a future refactor for NP.